addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -417,11 +417,34 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
     }
 }
 
+@keyframes vertical-marquee {
+    0% { // Spend more time on top positions
+        transform: translateY(0);
+    }
+    98% {  // Scroll back up so viewers are not lost
+        transform: translateY(var(--surveyLeaderboardTranslateY));  // defined in JS
+    }
+    100% {
+        transform: translateY(0);
+    }
+}
+
 .o_survey_session_leaderboard {
     font-size: 1.4rem;
 
```

Comment:
The score board is hard to read on the burger quiz background (almost black on black). We could improve it by adding on o_survey_session_leaderboard:
background-color: rgba(255, 255, 255, 0.5); border-radius: 10px; padding: 20px;

What do you think?